### DISCUSSION TITLE
Adding support for ppc64le architecture

### DISCUSSION THREAD

**Discussion by snehagg24** [concourse/concourse#8287] [Help & Support]
I am new to Concourse and I am currently exploring its functionality on x86 arch. Using docker-compose, I was able to easily setup the CI and add external workers as well. I would like to work on adding support for ppc64le arch. 
Concourse seems to have lots of components and the official CI looks bit complex to understand for beginners.
Can someone give me pointers on what would be the correct flow of components that I would need to update to add ppc64le support officially?
Currently, I am trying to build `concourse/concourse` image by using [this](https://github.com/concourse/concourse-docker/blob/master/Dockerfile) Dockerfile. Any pointers on how to build `linux-rc` which is a dependency for concourse image?

**Comment by snehagg24** [comment]
@taylorsilva @vito can you provide some pointers on this? Or point me to someone who can provide me some info?

**Reply by taylorsilva** [reply]
This slightly old blog post may be helpful: https://ops.tips/blog/raspberry-pi-concourse-ci-worker/
and Robin's work on the arm version of concourse: https://github.com/robinhuiser/concourse-arm64
